bluetooth/src/commonMain/kotlin/device/DeviceInfo.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright (c) 2020. Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.bluetooth.device
+
+import kotlin.math.pow
+
+expect class Identifier
```

Comment:
Can be protocol or have some contractor to use in tests?

Reply:
now there is the `uuidFrom` common method.